packages/web/src/scss/components/Tooltip/index.html

```diff
@@ -328,52 +328,88 @@ <h2 class="docs-Heading">Dismissible Tooltip via JS API</h2>
 
   </section>
 
-  <section id="advanced-positioning" class="docs-Section">
+  <section id="full-floating-ui" class="docs-Section spirit-feature-tooltip-enable-data-placement">
 
-    <h2 class="docs-Heading">Advanced Positioning</h2>
+    <h2 class="docs-Heading">Full Floating UI Usage</h2>
 
-    <div class="docs-Stack docs-Stack--start">
+    <div class="docs-Stack docs-Stack--stretch">
 
-      <p>
+      <p class="mb-0">
         The following example is using external library <a href="https://floating-ui.com">Floating UI</a>.
+        Try scrolling the frame or resizing the window to see how the Tooltip behaves. The Floating UI
+        library is trying to keep the Tooltip in the viewport and it is also flipping, shifting and
+        resizing the Tooltip when it is not possible to keep it in the viewport.
       </p>
 
-      <p>
-        🖱 Try scrolling the example to see how Tooltip placement is updated.
-      </p>
+      <div class="Alert Alert--informative" role="alert">
+        <svg width="24" height="24">
+          <use xlink:href="/icons/svg/sprite.svg#info" />
+        </svg>
+        <div>
+          Please note that the Floating UI library is trying to point the arrow to the center
+          of the trigger element. This is not possible to achieve in CSS only so our behavior
+          is slightly different.
```

Comment:
```suggestion
          Please note that the Floating UI library is trying to point the arrow to the center
          of the trigger element. This is not possible to achieve in CSS only so our behavior
          is slightly different for tooltips not using Floating UI.
```

Just to make it clearer…